### ISSUE TITLE
Stop time tracker automatically when the user log-out

### ISSUE THREAD

**Issue by ctchtc** [hicknhack-software/redmine_hourglass#78]
It would be very useful to stop a time tracker of an user when it logout. Normally when you logout you stopped working and the tracker should be automatically stopped. When you login again you have to choose manually what are you working at.

**Comment by slashworld** [comment]
IMHO the Redmine is not working place. I may start working and set Start in Redmine and close any browsers and for example connecting to the server which I really working by customer's ticket or any somewhere work.
Requesting feature might being implemented but not by default or any stricted style.

**Comment by arBmind** [comment]
@slashvlz I agree with your assessment. Often it would be helpful to store a hint, when the user actively logs out or closes the browser, that would give the users options. It would also be great to limit the timer. Tracking >12 hours seems wrong for most users.
